lib-api/src/main/java/org/ergoplatform/appkit/UnsignedTransaction.java

```diff
@@ -31,4 +27,10 @@ public interface UnsignedTransaction extends Transaction {
      * Returns the change address associated with this unsigned transaction
      */
     ErgoAddress getChangeAddress();
+
+    /**
+     * Optional list of tokens requested for burning in this transaction.
```

Comment:
Would be good to make it a bit more clear that this returns the tokens marked to burn, not the actual burned tokens. 

Maybe another method name would be better, as this one can easily misunderstood to return the actual tokens burned.

Also, `@Nonnull` annotation would be good to add here to make clear that the optional list is not returning null, but empty when there are no tokens requested to burn.

Reply:
Improved description and added annotation.
The name is consistent with boxesToSpend, inputsToSpend. Also `ToBurn` means in future, and the transaction not yet signed.
Please suggest a better name if you have some in mind.

Reply:
I think i is clear enough with the new JavaDoc!